analyses/mdg/mdg_plague_exploration.md

```diff
@@ -122,6 +147,21 @@ plague_path_old = plague_dir / plague_data_filename_old
 df=preprocess_plague_data(plague_path,list_cases_class=incl_cases_class,delimiter=",")
 ```
 
+```python
+# THIS DATA OUTDATED BY NEW DATA PROVIDED IN 2021-10-18 FILE
+# bring in 2012 to 2016 data for analysis
+#plague_path_2012_2016 = plague_dir / "Madagascar_IPM_Plague_cases_Aggregated_historic_2021-10-13.csv"
+#df_2012_2016 = preprocess_plague_data(plague_path_2012_2016,list_cases_class=incl_cases_class,delimiter=";")
```

Comment:
Since you commented out, the last part of the notebook on the comparison is not working anymore. Maybe load the old file in that last part instead?